package.json

```diff
@@ -22,13 +22,13 @@
   "repository": "CloudNativeJS/cloud-health",
   "homepage": "http://CloudNativeJS.io/",
   "devDependencies": {
-    "coveralls": "^3.0.2",
-    "typescript": "^3.3.3333",
     "@types/chai": "^4.0.5",
     "@types/mocha": "^5.2.6",
     "@types/node": "^12.0.0",
     "chai": "^4.1.2",
+    "coveralls": "^3.0.2",
+    "mocha": "^6.2.0",
     "nyc": "^14.1.0",
-    "mocha": "^6.0.2"
+    "typescript": "^3.3.3333"
```

Comment:
Maybe try going to the latest TypeScript version.

Reply:
Updated typescript and tests all pass